spec/request/session/create_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe 'POST #create sign_in', type: :request do
+  let!(:user_attrs) { { user: FactoryBot.attributes_for(:user) } }
+
+  subject do
+    post '/api/v1/users/sign_in', params: {
```

Comment:
You can use something like: `new_user_session_path` check the routes, and use the name prefix with `_path`